### PR TITLE
SEO-187924-ASP.NET-Web-Forms-PdfViewer-HTTP-to-HTPPS

### PR DESCRIPTION
**Pull request by Faith-Atieno**
Title | Description
-- | --
|Task Category | Site Audit work|
|Content Task Link/Mail Screenshot |
image
|
|Hotfix |https://github.com/syncfusion-content/asp.netwebforms-docs/pull/534|
|Ticket/Task link |https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187924|
|Work link |https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7B0C07BBB1-1233-47A7-818C-275A98FD0063%7D&file=Faith-%20Site%20Audit.xlsx&action=default&mobileredirect=true|

|Changes Made | Reason for change |
|-- | -- |
|Changed HTTP links to HTTPS | To avoid redirect links | 
|Modified | NA |
|Modified description | to improve user experience |

### CODE REVIEW

**Comment by Faith-Atieno** [comment]
rebuild

**Comment by Faith-Atieno** [comment]
rebuild

**Comment by Faith-Atieno** [comment]
rebuild

**Comment by Faith-Atieno** [comment]
rebuild

**Comment by Faith-Atieno** [comment]
rebuild

**Comment by Faith-Atieno** [comment]
rebuild

File: aspnet/DropDownList/DataBinding.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
 layout: post
 title: Data binding in DropDownList | Syncfusion | ASP.NET WebForms
-description: Describes about the data binding in DropDownList control for Syncfusion ASP.NET WebForm
+description: This article describes about the data binding in the DropDownList control for the Syncfusion ASP.NET WebForm
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/DataBinding.md:4]: The modified description isn't correct

File: aspnet/DropDownList/DataBinding.md [hunk 2]
```diff
@@ -379,7 +379,7 @@ It loads the data on scrolling the list of items. This can be achieved by settin
     {% highlight c# %}
         protected void Page_Load(object sender, EventArgs e)
         {
-            DropDownList1.DataSource = "http://mvc.syncfusion.com/Services/Northwnd.svc/Orders";
+            DropDownList1.DataSource = "https://ej2.syncfusion.com/home/javascript.html";
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/DataBinding.md:382]: Don't change links in code example

File: aspnet/DropDownList/Functionalities.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
 title: Functionalities in DropDownList | Syncfusion | ASP.NET WebForms
-description: Functionalities in the DropDownList control for Syncfusion ASP.NET WebForm
+description: This article explains more about ASP.NET Web Forms DropDownList Functionalities in the DropDownList control for Syncfusion ASP.NET WebForm
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/Functionalities.md:4]: Check all the description once

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DropDownList/Functionalities.md (hunk 3 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/Functionalities.md:4]: resolved

File: aspnet/DropDownList/DataBinding.md (hunk 2 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/DataBinding.md:382]: Changes reverted hence closing

**Comment by Faith-Atieno** [comment]
rebuild

File: aspnet/DropDownList/DataBinding.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
 ---
 layout: post
 title: Data binding in DropDownList | Syncfusion | ASP.NET WebForms
-description: Describes about the data binding in DropDownList control for Syncfusion ASP.NET WebForm
+description: Learn here about data binding with Syncfusion Essential ASP.NET Web Forms dropdownList control, its elements, and more.
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/DataBinding.md:4]: Check description

File: aspnet/DropDownList/DataBinding.md [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
 platform: aspnet
 control: DropDownList
 documentation: ug
 keywords: DropDownList, dropdown, data binding, Local data, Remote data
 ---
 
-# Data Binding
+# ASP.NET Web Forms DropDownList Data Binding
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/DataBinding.md:11]: Check H1

File: aspnet/DropDownList/DataBinding.md (hunk 2 above)
Comment (Yvone-Atieno) [inline aspnet/DropDownList/DataBinding.md:382]: As I told you. Don't make changes is code example, revert this

File: aspnet/DropDownList/Functionalities.md (hunk 3 above)
Comment (Yvone-Atieno) [inline aspnet/DropDownList/Functionalities.md:4]: This not an article, check and update description well

File: aspnet/DropDownList/Functionalities.md [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
 platform: aspnet
 control: DropDownList
 documentation: ug
 keywords: DropDownList, dropdown, Selection, Grouping, Sorting
 ---
-# Functionalities
+# ASP.NET Web Forms DropDownList Functionalities
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/Functionalities.md:10]: Please check this PR once again

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DropDownList/DataBinding.md (hunk 4 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/DataBinding.md:4]: resolved

File: aspnet/DropDownList/DataBinding.md (hunk 5 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/DataBinding.md:11]: resolved

File: aspnet/PdfViewer/GettingStarted.md [hunk 7]
```diff
@@ -62,7 +62,7 @@ Add the script files and CSS files in the &#60;title&#62; tag of Default.aspx.
 <script src="https://cdn.syncfusion.com/16.1.0.24/js/web/ej.web.all.min.js"></script>
 {% endhighlight %}
 
-N> For getting started, you can use the ej.web.all.min.js file, which encapsulates all the ej widgets and frameworks in one single file. In production, we highly recommend you to use our [custom script generator](https://help.syncfusion.com/js/include-only-the-needed-widgets# "custom script generator") to create custom script file with required widgets and its dependencies, to reduce the size of the script file.
```
Comment (Yvone-Atieno) [inline aspnet/PdfViewer/GettingStarted.md:65]: Why is this link removed

File: aspnet/PdfViewer/GettingStarted.md [hunk 8]
```diff
@@ -50,9 +50,9 @@ N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial se
 **Add Scripts and Styles**
 
 You can find the scripts for PDF viewer control and style sheets in any of the following locations:
-Local Disk: [Click here](https://help.syncfusion.com/js/installation-and-deployment# "Installation and deployment") to know more about script and style sheets installed in local machine.
-CDN Link: [Click here](https://help.syncfusion.com/js/cdn# "CDN scripts and styles links") to know more about script and style sheets available online.
-NuGet Package: [Click here](https://help.syncfusion.com/js/installation-and-deployment#configuring-syncfusion-nuget-packages "Configuring Syncfusion NuGet Packages") to know more about script and style sheets available in NuGet package.
```
Comment (Yvone-Atieno) [inline aspnet/PdfViewer/GettingStarted.md:55]: Why are this links removed

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DropDownList/Functionalities.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Faith-Atieno) [reply aspnet/DropDownList/Functionalities.md]: Resolved

File: aspnet/DropDownList/Functionalities.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Faith-Atieno) [reply aspnet/DropDownList/Functionalities.md]: Resolved

File: aspnet/PdfViewer/GettingStarted.md [hunk 9]
```diff
@@ -322,7 +322,7 @@ Add below code snippet to Default.aspx. Here, PDF viewer uses the Web API to pro
 <%@ Page Language="C#" AutoEventWireup="true" CodeBehind="Default.aspx.cs" Inherits="PdfViewerEJWeb.Default" %>
 <%@ Register assembly="Syncfusion.EJ.Web" namespace="Syncfusion.JavaScript.Web" tagprefix="ej" %>
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml">
+<html xmlns="https://www.w3.org/1999/xhtml/">
```
[the remark this answers is not in the document]
Comment (Faith-Atieno) [reply aspnet/PdfViewer/GettingStarted.md:325]: Resolved

File: aspnet/PdfViewer/GettingStarted.md (hunk 7 above)
Comment (Faith-Atieno) [reply aspnet/PdfViewer/GettingStarted.md:65]: resolved

File: aspnet/PdfViewer/GettingStarted.md (hunk 8 above)
Comment (Faith-Atieno) [reply aspnet/PdfViewer/GettingStarted.md:55]: Linked returned

File: aspnet/DropDownList/DataBinding.md [hunk 10]
```diff
@@ -1,16 +1,16 @@
 ---
 layout: post
-title: Data binding in DropDownList | Syncfusion | ASP.NET WebForms
-description: Describes about the data binding in DropDownList control for Syncfusion ASP.NET WebForm
+title: Data binding in ASP.NET WebForms Dropdown List | Syncfusion
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/DataBinding.md:3]: Use ASP.NET Web Forms

File: aspnet/DropDownList/DataBinding.md [hunk 11]
```diff
@@ -404,7 +404,7 @@ N> In both modes, set of items will be fetched based on the count specified in t
     {% highlight c# %}
         protected void Page_Load(object sender, EventArgs e)
         {
-            DropDownList1.DataSource = "http://mvc.syncfusion.com/Services/Northwnd.svc/Orders";
+            DropDownList1.DataSource = "https://ej2.syncfusion.com/home/javascript.html";
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/DataBinding.md:407]: Revert this change

File: aspnet/DropDownList/Functionalities.md [hunk 12]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Functionalities in DropDownList | Syncfusion | ASP.NET WebForms
-description: Functionalities in the DropDownList control for Syncfusion ASP.NET WebForm
+title: Functionalities in ASP.NET Web Forms Dropdown List | Syncfusion
+description: Learn here all about Functionalities in Syncfusion ASP.NET Web Forms Dropdown List component of Syncfusion Essential JS 2 and more.
 platform: aspnet
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/Functionalities.md:5]: Learn here all about functionalities in Syncfusion ASP.NET Web Forms Dropdown List control, it's elements, and more.

File: aspnet/DropDownList/Functionalities.md [hunk 13]
```diff
@@ -305,7 +305,7 @@ I> To use "SelectedIndices" property, you should enable either ShowCheckbox or M
 
 ### Unselect items
 
-Similarly, you can unselect a single or multiple items by using [unselectItemByValue](http://help.syncfusion.com/js/api/ejdropdownlist#methods:unselectitembyvalue) or [unselectItemByIndices](http://help.syncfusion.com/js/api/ejdropdownlist#methods:unselectitembyindices) or [unselectItemByText](http://help.syncfusion.com/js/api/ejdropdownlist#methods:unselectitembytext) methods. This will remove the selection state of the corresponding data item from the popup list and textbox. 
+Similarly, you can unselect a single or multiple items by using [unselectItemByValue](https://help.syncfusion.com/api/js/ejdropdownlist#methods:unselectitembyvalue)) or [unselectItemByIndices](http://help.syncfusion.com/js/api/ejdropdownlist#methods:unselectitembyindices) or [unselectItemByText](https://help.syncfusion.com/api/js/ejdropdownlist#methods:unselectitembytext) methods. This will remove the selection state of the corresponding data item from the popup list and textbox.
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/Functionalities.md:308]: Why is one link not changed yet is redirecting

File: aspnet/DropDownList/Validation.md [hunk 14]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Validation in DropDownList control for Syncfusion ASP.NET WebForm
-description: Describes about the Validation in DropDownList control for Syncfusion ASP.NET WebForm
+title: Validation in  ASP.NET WebForm Dropdown List control | Syncfusion
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/Validation.md:3]: Update as Validation in ASP.NET Web Forms Dropdown List Control | Syncfusion

File: aspnet/DropDownList/Validation.md [hunk 15]
```diff
@@ -1,15 +1,15 @@
...
+description: Learn here all about ports support in Syncfusion Essential ASP.NET Webforms Dropdown List Control, its elements, and more.
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/Validation.md:4]: Update platform name well and it should be control not Control

File: aspnet/DropDownList/Validation.md [hunk 16]
```diff
@@ -1,15 +1,15 @@
...
 platform: aspnet
 control: DropDownList
 documentation: ug
 keywords: Validation, DropDownList, dropdown, jQuery Validator, ASP Validator
 
 ---
 
-# Validation
+# Validation in  ASP.NET Web Form Dropdown List control
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/Validation.md:12]: Remove extra space and use Control not control

File: aspnet/DropDownList/Validation.md [hunk 17]
```diff
@@ -19,7 +19,7 @@ DropDownList value can be validated in two ways,
  
 You can validate the DropDownList value on form submission using jQuery Validator, by applying “ValidationRules” and “ValidationMessage” to the DropDownList. 
 
-N> [jquery.validate.min](http://cdn.syncfusion.com/js/assets/external/jquery.validate.min.js) script file should be referred for validation, for more details, refer [here](http://jqueryvalidation.org/documentation).
```
Comment (Yvone-Atieno) [inline aspnet/DropDownList/Validation.md:22]: Why is this link removed?

File: aspnet/PdfViewer/GettingStarted.md [hunk 18]
```diff
@@ -8,7 +8,7 @@ documentation: ug
 ---
 
 
-# Getting Started
+# Getting Started with ASP.NET Webforms PDF viewer
```
Comment (Yvone-Atieno) [inline aspnet/PdfViewer/GettingStarted.md:11]: Check platform name in all places and update correctly

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DropDownList/DataBinding.md (hunk 10 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/DataBinding.md:3]: resolved

File: aspnet/DropDownList/Functionalities.md (hunk 12 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/Functionalities.md:5]: resolved

File: aspnet/DropDownList/Functionalities.md (hunk 13 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/Functionalities.md:308]: resolved

File: aspnet/DropDownList/Validation.md (hunk 14 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/Validation.md:3]: resolved

File: aspnet/DropDownList/Validation.md (hunk 16 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/Validation.md:12]: resolved

File: aspnet/DropDownList/Validation.md (hunk 17 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/Validation.md:22]: link returned

File: aspnet/DropDownList/DataBinding.md (hunk 11 above)
Comment (Faith-Atieno) [reply aspnet/DropDownList/DataBinding.md:407]: previous link returned

**Comment by Faith-Atieno** [comment]
resolved

**Comment by Faith-Atieno** [comment]
rebuild

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DropDownList/Functionalities.md [hunk 19]
```diff
@@ -1,13 +1,13 @@
...
-description: Functionalities in the DropDownList control for Syncfusion ASP.NET WebForm
+title: Functionalities in ASP.NET Web Forms Dropdown List | Syncfusion
+description: Learn here all about functionalities in Syncfusion ASP.NET Web Forms Dropdown List control, it's elements, and more.
 platform: aspnet
 control: DropDownList
 documentation: ug
 keywords: DropDownList, dropdown, Selection, Grouping, Sorting
 ---
-# Functionalities
+# Functionalities in ASP.NET Web Forms Dropdown List Component
```
[the remark this answers is not in the document]
Comment (Faith-Atieno) [reply aspnet/DropDownList/Functionalities.md:10]: resolved

File: aspnet/PdfViewer/GettingStarted.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (Faith-Atieno) [reply aspnet/PdfViewer/GettingStarted.md:65]: link returned

File: aspnet/PdfViewer/GettingStarted.md [hunk 20]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting Started | PDF viewer | ASP.NET Webforms | Syncfusion
-description: Learn here about getting started with Syncfusion ASP.NET Webforms PDF viewer control and more details.
+title: Getting Started | PDF Viewer | ASP.NET Web Forms | Syncfusion
+description: Learn here about getting started with Syncfusion ASP.NET Web Forms PDF Viewer control and more details.
```
[the remark this answers is not in the document]
Comment (Faith-Atieno) [reply aspnet/PdfViewer/GettingStarted.md]: resolved

File: aspnet/PdfViewer/GettingStarted.md [hunk 21]
```diff
@@ -45,14 +45,14 @@ Add below references, which are necessary for the PDF viewer control, to the pro
...
 
 **Add Scripts and Styles**
 
 You can find the scripts for PDF viewer control and style sheets in any of the following locations:
-Local Disk: [Click here](https://help.syncfusion.com/js/installation-and-deployment# "Installation and deployment") to know more about script and style sheets installed in local machine.
-CDN Link: [Click here](https://help.syncfusion.com/js/cdn# "CDN scripts and styles links") to know more about script and style sheets available online.
-NuGet Package: [Click here](https://help.syncfusion.com/js/installation-and-deployment#configuring-syncfusion-nuget-packages "Configuring Syncfusion NuGet Packages") to know more about script and style sheets available in NuGet package.
+Local Disk: [Installation and deployment](https://help.syncfusion.com/js/overview) to know more about script and style sheets installed in local machine.
+CDN Link: [CDN scripts and styles links](https://help.syncfusion.com/js/cdn#) to know more about script and style sheets available online.
+NuGet Package: [Configuring Syncfusion.uGet Packages](https://help.syncfusion.com/js/overview) to know more about script and style sheets available in NuGet package.
```
[the remark this answers is not in the document]
Comment (Faith-Atieno) [reply aspnet/PdfViewer/GettingStarted.md]: resolved

File: aspnet/PdfViewer/GettingStarted.md [hunk 22]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting Started | PDF viewer | ASP.NET Webforms | Syncfusion
-description: Learn here about getting started with Syncfusion ASP.NET Webforms PDF viewer control and more details.
+title: Getting Started | PDF Viewer | ASP.NET Web Forms | Syncfusion
```
Comment (Yvone-Atieno) [inline aspnet/PdfViewer/GettingStarted.md:3]: Update proper title

File: aspnet/PdfViewer/GettingStarted.md [hunk 23]
```diff
@@ -62,7 +62,7 @@ Add the script files and CSS files in the &#60;title&#62; tag of Default.aspx.
 <script src="https://cdn.syncfusion.com/16.1.0.24/js/web/ej.web.all.min.js"></script>
 {% endhighlight %}
 
-N> For getting started, you can use the ej.web.all.min.js file, which encapsulates all the ej widgets and frameworks in one single file. In production, we highly recommend you to use our [custom script generator](https://help.syncfusion.com/js/include-only-the-needed-widgets# "custom script generator") to create custom script file with required widgets and its dependencies, to reduce the size of the script file.
+N> For getting started, you can use the ej.web.all.min.js file, which encapsulates all the ej widgets and frameworks in one single file. In production, we highly recommend you to use our [custom script generator](https://help.syncfusion.com/js/overview) to create custom script file with required widgets and its dependencies, to reduce the size of the script file.
```
Comment (Yvone-Atieno) [inline aspnet/PdfViewer/GettingStarted.md:65]: Update proper link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/PdfViewer/GettingStarted.md (hunk 22 above)
Comment (Faith-Atieno) [reply aspnet/PdfViewer/GettingStarted.md:3]: resolved

File: aspnet/PdfViewer/GettingStarted.md (hunk 23 above)
Comment (Faith-Atieno) [reply aspnet/PdfViewer/GettingStarted.md]: resolved

Comment (Faith-Atieno) [reply aspnet/PdfViewer/GettingStarted.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

File: aspnet/DropDownList/Functionalities.md [hunk 24]
```diff
@@ -97,7 +97,7 @@ The following methods, select the items in the DropDownList.
     </tr>
     <tr>
         <td>
-            {{'[selectItemByText](http://help.syncfusion.com/js/api/ejdropdownlist#methods:selectItemByText)'| markdownify }}
+            {{'[selectItemByText](https://help.syncfusion.com/api/js/ejdropdownlist#methods:selectItemByText)'| markdownify }}
```
Comment (Aishwarya-Ganesan) [inline aspnet/DropDownList/Functionalities.md:100]: why you changed inside the code snippet?

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented.
